src/main/resources/assets/api-docs.yaml

```diff
@@ -2973,6 +2973,164 @@ paths:
+    post:
+      summary: Library Card CREATE resource
+      description: Creates a new Library Card based on user-provided json
+      parameters:
+        - name: LibraryCard
+          in: body
+          description: JSON representation of libraryCard payload
+          required: true
+          schema:
+            $ref: '#/components/schemas/LibraryCard'
+      tags:
+        - Library Card
+        - Admin
+      responses:
+        200:
+          description: Library Card CREATE success
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/LibraryCard'
+        400:
+          description: Bad request (Issues with json payload)
+        401:
+          description: User not authorized
+        409:
+          description: Library Card payload conflicts with an existing record
+        500:
+          description: Internal server error
+  /api/libraryCards/{id}:
+    get:
+      summary: Library Card GET Resource
+      description: Fetch library card bases on id parameter
+      tags:
+        - Library Card
+        - Admin
+      parameters:
+      - name: id
+        in: path
+        description: Library Card ID
+        required: true
+        schema:
+          type: integer
+      responses:
+        200:
+          description: Library Card GET success
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/LibraryCard'
+        401:
+          description: User not authorized
+        404:
+          description: Library Card not found
+        500:
+          description: Internal server error
+    put:
+      summary: Library Card PUT resource
+      description: Update target Library Card based on id parameter and user provided json
+      tags:
+        - Library Card
+        - Admin
+      parameters:
+      - name: id
+        in: path
+        description: Library Card ID
+        required: true
+        schema:
+          type: integer
+      - name: libraryCard
+        in: body
+        description: JSON representation of Library Card
+        required: true
+        schema:
+          $ref: '#/components/schemas/LibraryCard'
+      responses: 
+        200:
+          description: Library Card PUT request success
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/LibraryCard'
+        400:
+          description: Bad request (Issues with json payload)
+        401:
+          description: User not authorized
+        404:
+          description: Library Card not found
+        409:
+          description: Library Card payload conflicts with an existing record
+        500:
+          description: Internal server error
+    delete:
+      summary: Library Card DELETE resource
+      description: Delete target Library Card based on id parameter
+      tags:
+        - Library Card
+        - Admin
```

Comment:
This is a nice addition, but begs the question of whether we should add this tag to all of the other admin-endpoints. Also, it could be expanded to Researcher, DAC Chair/Member, Data Custodian, etc. Let's have a team discussion about whether we want to go down this path. This should not hold up the PR.